packages/airnode-protocol/hardhat.config.js

```diff
@@ -23,5 +23,11 @@ module.exports = {
   },
   solidity: {
     version: '0.8.9',
```

Comment:
I'm also making the assumption that 0.8.0 is the highest patch version that inheritance permits right? I'm guessing that versions form 0.8.1 to 0.8.8 won't work

Reply:
I used `^0.8.0` in interfaces and contracts to be inherited and `0.8.9` in contracts that will be deployed. This is because we will be using `0.8.9`, but we don't want to lock the user into that if they will be importing `@api3/airnode-protocol` to use some of these contracts to build their own project. For example, the user is expected to inherit `RrpRequester` but if the version of that is `0.8.9` then the requester contract will also have to be exactly `0.8.9`, which is annoying for the user.